pkg/identityserver/contact_info_registry_test.go

```diff
@@ -48,9 +48,9 @@ func TestContactInfoValidation(t *testing.T) {
 	testWithIdentityServer(t, func(is *IdentityServer, cc *grpc.ClientConn) {
 		reg := ttnpb.NewContactInfoRegistryClient(cc)
 
-		retryInterval := test.Delay << 3
+		retryInterval := test.Delay << 7 // default of 128 ms
```

Comment:
Is it 128 ms under `TEST_DELAY=8` ?

Reply:
well the default is `1 * time.Millisecond` on the test.Delay, but indeed on the CI is `8 * time.Millisecond`.
I can decrease the value a bit, because indeed under the CI value it would delay the whole test by a second or so.

Reply:
The one extra second is fine, but what I meant is that you should test this with `TEST_DELAY=8` and see where things go bad, before increasing the timeouts themselves.

Reply:
I'll remove the comment then and I'll test it with a smaller value, something like `test.Delay * (2^5)`

Reply:
>The one extra second is fine, but what I meant is that you should test this with TEST_DELAY=8 and see where things go bad, before increasing the timeouts themselves.

Okay, will do